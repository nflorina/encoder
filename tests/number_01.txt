12
-18396
1171
-15678159
1955274
-13312812
-134
117188
162
-1941
-18
-1287
152
1399757
-129455
1869
-19384
11449557
1526199
-131937
161751
17999751
-15119421
1747
-117228
-11
-12558
-1495
111
-1871765
1245
18741553
-19112988
-14928466
127311
-191992
-16663941
19978
-149
1755
165141
14
19925
18111114
-1446
1817
-117
1233245
-1139956
-1319
1
END
