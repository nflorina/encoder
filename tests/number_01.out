0 0 50
2118396711115678159955274112133128134881171621119418128752199757131294559186193849557114499152611319377511619997511711511942747111722811125581495111176518751248741553111298819149284667311121919921166639499781149755165141141992518111114114468171117512332411399561319
2118396711115678159955274112133128134881171621119418128752199757131294559186193849557114499152611319377511619997511711511942747111722811125581495111176518751248741553111298819149284667311121919921166639499781149755165141141992518111114114468171117512332411399561319
